R/internals.R

```diff
@@ -1,3 +1,29 @@
+.rename_fb_cols <- function(df) {
+  var_names <- df[1,] %>% as.character()
+
+  new_names <- paste(var_names, names(df), sep = "_")
+
+  new_names <- new_names %>%
+    gsub("\\..[0-9]", "", .) %>%
+    gsub("\\.[0-9]", "", .) %>%
+    gsub("\\.", "_", .) %>%
+    gsub("_Var", "", .) %>%
+    gsub("# Pl", "Num_Players", .) %>%
+    gsub("%", "_percent", .) %>%
+    gsub("_Performance", "", .) %>%
+    # gsub("_Penalty", "", .) %>%
+    gsub("1/3", "Final_Third", .) %>%
+    gsub("\\+/-", "Plus_Minus", .) %>%
+    gsub("/", "_per_", .) %>%
+    gsub("-", "_minus_", .) %>%
+    gsub("90s", "Mins_Per_90", .) %>%
+    gsub("__", "_", .) %>%
+    gsub("_$", "", .)
```

Comment:
added this last part because i was getting column names like `Squad_` without this